### ISSUE TITLE
Cannot read property 'substring' of undefined

### ISSUE THREAD

**Issue by atian25** [popomore/coffee#26]
```
/usr/local/bin/node /Users/tz/Workspaces/ali/nut/node_modules/mocha/bin/_mocha --ui bdd --reporter /Applications/WebStorm.app/Contents/plugins/NodeJS/js/mocha-intellij/lib/mochaIntellijReporter.js /Users/tz/Workspaces/ali/nut/test/index.test.js
module.js:216
  var start = request.substring(0, 2);
                     ^

TypeError: Cannot read property 'substring' of undefined
    at Function.Module._resolveLookupPaths (module.js:216:22)
    at Function.Module._resolveFilename (module.js:329:31)
    at Function.require.resolve (module.js:389:19)
    at findIstanbul (/Users/tz/Workspaces/ali/nut/node_modules/@ali/mm/node_modules/coffee/lib/find_istanbul.js:15:26)
    at Object.<anonymous> (/Users/tz/Workspaces/ali/nut/node_modules/@ali/mm/node_modules/coffee/index.js:8:33)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
```

**Comment by atian25** [comment]
run mocha in terminal ok, but run in webstorm mocha will fail

**Comment by popomore** [comment]
Fixed in #27
